liberation fonts: updates

dtzWill opened the pull request:
###### Motivation for this change


###### Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

dtzWill committed:
liberation_ttf: 2.00.1 -> 2.00.4

(both binary and source versions)

Upstream moved to github, release URL's are less simple.

dtzWill committed:
liberation fonts v1: 1.07.4 -> 1.07.5

dtzWill committed:
liberationsansnarrow{,_binary}: 1.07.3 -> 1.07.6

dtzWill committed:
liberation fonts: update homepage to new location (github)

volth commented on pkgs/data/fonts/redhat-liberation-fonts/default.nix:
```diff
@@ -50,36 +50,36 @@ let
-    url = "https://releases.pagure.org/liberation-fonts/liberation-fonts-${version}.tar.gz";
-    sha256 = "01jlg88q2s6by7qv6fmnrlx0lwjarrjrpxv811zjz6f2im4vg65d";
+    version = "1.07.5";
+    url = https://github.com/liberationfonts/liberation-1.7-fonts/files/2175696/liberation-fonts-1.07.5.tar.gz;
```
maybe ```fetchFromGitHub { ...; rev = version; }``` for source versions ?
Repos have release tags (https://github.com/liberationfonts/liberation-1.7-fonts/releases https://github.com/liberationfonts/liberation-fonts/releases)

worldofpeace commented:
@volth Any reason why we need binary and source packages?

volth commented:
> @volth Any reason why we need binary and source packages?

Binary could be removed

worldofpeace commented:
> > @volth Any reason why we need binary and source packages?
> 
> Binary could be removed

Did the thing https://github.com/NixOS/nixpkgs/pull/51331/commits/0e822cb345bc79bed67af4fbd041455a073ffa23

worldofpeace commented on pkgs/top-level/all-packages.nix:
```diff
@@ -15382,11 +15382,10 @@ with pkgs;
-  liberation_ttf = liberation_ttf_v2_binary;
+    liberation_ttf_v1
+    liberation_ttf_v2
+    ;
```
The placement of this semicolon is purposeful.

worldofpeace committed:
liberationsansnarrow_binary: remove

worldofpeace committed:
liberationsansnarrow: use fetchFromGitHub

worldofpeace committed:
liberationsansnarrow: rename to liberation-sans-narrow

worldofpeace committed:
liberation fonts: drop binary packages

That means the following packages  are removed:
  - liberation_ttf_v1_binary
  - liberation_ttf_v2_binary

I've renamed these attributes because the binary form is removed:

  - liberation_ttf_v1_from_source -> liberation_ttf_v1
  - liberation_ttf_v2_from_source -> liberation_ttf_v2

Aliases are added for these.

worldofpeace committed:
liberation fonts: use fetchFromGitHub
